system-contracts/contracts/EvmEmulator.yul

```diff
@@ -6148,10 +6226,40 @@ object "EvmEmulator" {
                 }
             }
 
+            function $llvm_Cold_llvm$_delegate7702(
+                delegationAddress,
+            ) -> success, returnOffset, returnLen {
+                returnOffset := MEM_OFFSET()
+                // TODO: use delegatecall by reference to avoid copying calldata
+                let calldataSize := calldatasize()
+                calldatacopy(0, 0, calldataSize)
+                success := delegatecall(gas(), delegationAddress, 0, calldataSize, 0, 0)
```

Comment:
Dangerous: we rely on fact that users can't initiate system calls from EVM addresses. Should be double checked everywhere